app/views/components/heading.scala.html

```diff
@@ -14,11 +14,18 @@
  * limitations under the License.
  *@
 
-@(title: String, heading: String)(implicit messages: Messages)
+@(sectionHeader: Option[String] = None, title: Option[String], hint: Option[String] = None)
 
-<fieldset>
-    <legend class="heading-large" role="heading" aria="" level="1">
-        <span class="heading-secondary">@messages(title)</span>
-        <h1 style="margin:0">@messages(heading)</h1>
-    </legend>
-</fieldset>
\ No newline at end of file
+<header class="page-heading">
+    @if(sectionHeader.nonEmpty) {
+        <span class="visuallyhidden">This section is </span>
+        <span id="section-header" class="heading-secondary">@sectionHeader.get</span>
```

Comment:
Nit: To avoid using `.get` and to be more scala-stylish you could use `map` on `sectionHeader`